techniques/system/common/1.0/promises.st

```diff
@@ -68,7 +69,7 @@ bundle common va
 
       # On the policy server, we do the check_log_system at the end, to ensure that rsyslog has been installed and configured before.
 
-      "end" slist => { "check_log_system", "endExecution" };
+      "end" slist => { "restart_services", "endExecution" };
```

Comment:
restart_services is made only on policy_server, not on standart node

restart_services should always be executed